sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_models.py

```diff
@@ -3809,6 +3861,9 @@ class DocTypeInfo(object):
     fields and types, and the confidence for those fields.
 
     :ivar str description: A description for the model.
+    :ivar build_mode: The build mode used when building the custom model.
+     Possible values include: "template", "neural".
+    :vartype build_mode: str or :class:`~azure.ai.formrecognizer.DocumentBuildMode`
```

Comment:
this is an output model only, right? I think vartype should be str only. We can say in the ivar doc that its value is represented by DocumentBuildMode but the user won't ever get an enum returned here.